src/components/RegistrationForm.tsx

```diff
@@ -39,54 +40,72 @@ const RegistrationForm = ({ onValidSubmit } : RegistrationFormProps) : JSX.Eleme
         ev.preventDefault();
     };
 
+    const field1 = <Msg id="forms.reg.fields.firstName"/>;
+    const field2 = <Msg id="forms.reg.fields.lastName"/>;
+    const field3 = <Msg id="forms.reg.fields.email"/>;
+    const field4 = <Msg id="forms.reg.fields.phone"/>;
+    const field5 = <Msg id="forms.reg.fields.password"/>;
+
+    const messageToString = useIntl().formatMessage;
```

Comment:
Instead of just aliasing `formatMessage` by storing it in another variable, create a new function here, give it a short name (e.g. `msg`) and have it take the ID as an argument, passing that ID _inside an object_ to `formatMessage`.